fcrepo-kernel-impl/src/test/java/org/fcrepo/integration/kernel/impl/FedoraResourceImplIT.java

```diff
@@ -647,6 +647,24 @@ public void testGetChildrenWithBinary() {
     }
 
     @Test
+    public void testGetChildrenWithBinaryCustomProperties() throws RepositoryException {
+        final String pid = getRandomPid();
+        final Container container = containerService.findOrCreate(session, "/" + pid);
+        final FedoraResource resource = binaryService.findOrCreate(session, "/" + pid + "/a");
+        final javax.jcr.Node node = resource.getNode();
+        node.setProperty("dc:title", "this-is-some-title");
+
+        final Graph graph =
+                container.getChildren().next().getTriples(subjects, PropertiesRdfContext.class).asModel().getGraph();
```

Comment:
I am not assuming that the child will be the first child. A parent can have multiple children and there maybe just one  child with dc:title among them. I was not think of retrieve the child by name because the change I made in  PropertiesRdfContext.java was Iterators...